### PR TITLE
Updated for UE5 and Marketplace reference

### PR DESCRIPTION
**Pull request by microsoftman**
Added UE5 content reference, and added links to the recommended Marketplace solution

### CODE REVIEW

File: azure-gaming-pr/reference-architectures/unreal-pixel-streaming-deploying.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ ms.prod: azure-gaming
...
+This guide will walk through the process of taking the exported Unreal Pixel Streaming package and deploying it on an Azure GPU Virtual Machine, or multiple VMs if using the Matchmaker service to load balance across many streams on multiple GPU VMs. For a detailed overview of Pixel Streaming and architectures in Azure, see our documentation [here](unreal-pixel-streaming-in-azure.md). For deploying Unreal Pixel Streaming at scale (i.e., 2 to 1000&#39;s of streams) with the easy to deploy Azure Marketplace solution, see the documentation [here](https://docs.unrealengine.com/4.27/ProductionPipelines/CloudDeployments/AzurePixelStreaming/), which allows users to deploy Pixel Streaming in Azure right from the Azure Portal. This now supports Unreal Engine 4 and Unreal Engine 5 apps. See Epic Games' Azure [Marketplace Offer](https://azuremarketplace.microsoft.com/marketplace/apps/epicgames.unreal-pixel-streaming?tab=Overview).
```
Comment (joannaleecy) [inline azure-gaming-pr/reference-architectures/unreal-pixel-streaming-deploying.md]: Consider breaking your sentence up so it reads better.

"To learn how to deploy Unreal Pixel Streaming at scale, from two to thousands of streams, with the easy to deploy Azure Marketplace solution, see [Deploying Unreal Engine 4 Pixel Streaming on Azure](https://docs.unrealengine.com/4.27/ProductionPipelines/CloudDeployments/AzurePixelStreaming/). This marketplace solution allows users to deploy Pixel Streaming in Azure, right from the Azure Portal. It also supports both Unreal Engine 4 and 5 apps. For more information, see [Unreal Pixel Streaming in Azure marketplace](https://azuremarketplace.microsoft.com/marketplace/apps/epicgames.unreal-pixel-streaming?tab=Overview)."

File: azure-gaming-pr/reference-architectures/unreal-pixel-streaming-deploying.md [hunk 2]
```diff
@@ -52,22 +52,22 @@ Depending on your **Region** needs, choose a **Region** that is the closest to t
...
-13. In the Networking page on the left, click the [Add inbound port rule](/azure/virtual-machines/windows/nsg-quickstart-portal#create-an-inbound-security-rule) button to open up any additional ports such as 8888 (PixelStreamingPort between UE4 + SS), 9999 (if using MM), 443 (if needing Https) and any STUN/TURN server ports being used (e.g., 19302). STUN/TURN servers not _required_ for most users, though some mobile or restricted networks might require these—learn more [here](#turn-and-stun-servers). As always, please follow your security requirements to ensure desired network safety.
+13. In the Networking page on the left, click the [Add inbound port rule](/azure/virtual-machines/windows/nsg-quickstart-portal#create-an-inbound-security-rule) button to open up any additional ports such as 8888 (PixelStreamingPort between UE app + SS), 9999 (if using MM), 443 (if needing Https) and any STUN/TURN server ports being used (e.g., 19302). STUN/TURN servers not _required_ for most users, though some mobile or restricted networks might require these—learn more [here](#turn-and-stun-servers). As always, please follow your security requirements to ensure desired network safety.
```
Comment (joannaleecy) [inline azure-gaming-pr/reference-architectures/unreal-pixel-streaming-deploying.md]: Before using "UE", define the acronym on first use.

Unreal Engine (UE)

File: azure-gaming-pr/reference-architectures/unreal-pixel-streaming-deploying.md [hunk 3]
```diff
@@ -52,22 +52,22 @@ Depending on your **Region** needs, choose a **Region** that is the closest to t
...
 
 [![RDP Login](media/pixel-streaming/pixel-streaming-rdp-login.png)](media/pixel-streaming/pixel-streaming-rdp-login.png)
 
-15. Once connected to the VM, accept the Windows Privacy options and copy (right click -&gt; copy) and paste (right click -&gt; paste) the exported Pixel Streaming package ([Unreal docs](https://docs.unrealengine.com/SharingAndReleasing/PixelStreaming/PixelStreamingIntro/index.html)) to the Windows desktop of the VM (i.e., WindowsNoEditor folder contents). **Note:** To speed up the copy it&#39;s recommended to zip up the files before copying over.
+15. Once connected to the VM, accept the Windows Privacy options and copy (right click -&gt; copy) and paste (right click -&gt; paste) the exported Pixel Streaming package ([Unreal docs](https://docs.unrealengine.com/SharingAndReleasing/PixelStreaming/PixelStreamingIntro/index.html)) to the Windows desktop of the VM (i.e., WindowsNoEditor folder contents for UE4 apps, or Windows folder for UE5). **Note:** To speed up the copy it&#39;s recommended to zip up the files before copying over.
```
Comment (joannaleecy) [inline azure-gaming-pr/reference-architectures/unreal-pixel-streaming-deploying.md:63]: I know your change is just to update some details in the instructions.

Do you mean that developers should follow the instructions on this page to create the package?: [Unreal docs](https://docs.unrealengine.com/SharingAndReleasing/PixelStreaming/PixelStreamingIntro/index.html)

**Review by joannaleecy — CHANGES_REQUESTED** [review]

File: azure-gaming-pr/reference-architectures/unreal-pixel-streaming-deploying.md (hunk 1 above)
Comment (microsoftman) [reply azure-gaming-pr/reference-architectures/unreal-pixel-streaming-deploying.md]: Thank you. I'm removing the 4 out of the title of the docs, as this supports 5 now as well. Epic will be updating their Azure docs as well.

File: azure-gaming-pr/reference-architectures/unreal-pixel-streaming-deploying.md (hunk 3 above)
Comment (microsoftman) [reply azure-gaming-pr/reference-architectures/unreal-pixel-streaming-deploying.md:63]: Yes, I put this pre-requisite up at the top as well to make it more clear.

**Review by joannaleecy — APPROVED** [review]

**Pull request merged**